frontend/app/component/pollingstation/PollingStationFormNavigation.tsx

```diff
@@ -152,7 +152,7 @@ export function PollingStationFormNavigation({ pollingStationId, election }: Pol
         </Modal>
       )}
 
-      {error && <Feedback id="feedback-server-error" type="error" data={error} />}
+      {apiError && <Feedback id="feedback-server-error" type="error" apiError={apiError} />}
```

Comment:
It would be nice to have some whitespace below this error message.
![image](https://github.com/user-attachments/assets/1c6ea95b-69af-4e5f-b964-aa28fe4703c1)

Reply:
Fixed in 0c1467b884c6e32855ef6188c5cf8a793ed93650.